parser_library/src/debugging/set_symbol_variable.h

```diff
@@ -42,14 +42,15 @@ class set_symbol_variable : public variable
 
 private:
     template<typename T>
-    T get_value() const;
+    T get_value(const std::optional<int>& index) const;
```

Comment:
given the size, these could be by-value... but I don't mind.

Reply:
I don't mind changing it either.